chronolog.py

```diff
@@ -0,0 +1,88 @@
+import os.path
+import regex as re
+
+import ChronologBackend.nlp as nlp
+import argparse
+from argparse import RawTextHelpFormatter
+
+DEFAULT_SPLITS = 10
+
```

Comment:
Would be useful to add some comments explaining what these regexes are, i.e what string patterns they match with.